build/package/Installer.DEB.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Target Name="SetupDebProps"
-          DependsOnTargets="Init">
+      DependsOnTargets="Init">
```

Comment:
No strong opinion, this file has a lot of extra whitespace so I run xml format over the file to fix them.

I will revert it back